docs/gettingstarted/quickstart.ipynb

```diff
@@ -2,24 +2,175 @@
  "cells": [
```

Comment:
<p>change "in the small" to "from the small"   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>